cmd/start.go

```diff
@@ -95,6 +100,13 @@ func (c *startCfg) registerFlags(fs *flag.FlagSet) {
 		fetch.DefaultMaxChunkSize,
 		"the range for fetching blockchain data by a single worker",
 	)
+
+	fs.IntVar(
+		&c.rateLimit,
+		"http-rate-limit",
+		0,
+		"the maximum HTTP requests allowed per minute per IP, unlimited by default",
```

Comment:
Could you please add the flag/instructions to activate the rate limit on the readme file? 🙏

Reply:
Yep! thanks for the heads up!